README.md

```diff
@@ -41,145 +41,35 @@ See Github [Issues](https://github.com/kubernetes-sigs/gcp-compute-persistent-di
-$ ./deploy/kubernetes/deploy-driver.sh
-```
-
-### Zonal Example
-
-1. Create example Zonal Storage Class
-```
-$ kubectl apply -f ./examples/kubernetes/demo-zonal-sc.yaml
-```
-
-2. Create example PVC and Pod
-```
-$ kubectl apply -f ./examples/kubernetes/demo-pod.yaml
-```
-
-3. Verify PV is created and bound to PVC
-```
-$ kubectl get pvc
-NAME      STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
-podpvc     Bound     pvc-e36abf50-84f3-11e8-8538-42010a800002   10Gi       RWO            csi-gce-pd     9s
-```
-
-4. Verify pod is created and in `RUNNING` state (it may take a few minutes to get to running state)
-```
-$ kubectl get pods
-NAME                      READY     STATUS    RESTARTS   AGE
-web-server                1/1       Running   0          1m
-```
-
-### Snapshot Example
-
-Please note that VolumeSnapshot is currently alpha feature. In order to enable this feature, please install CSI driver with
-dev version by setting the environment variable "GCE_PD_DRIVER_VERSION=dev"
-1. Create example Default Snapshot Class
-```
-$ kubectl create -f ./examples/kubernetes/demo-defaultsnapshotclass.yaml
-```
-2. Create a snapshot of the PVC created in above example
-```
-$ kubectl create -f ./examples/kubernetes/demo-snapshot.yaml
-```
-3. Verify Snapshot is created and is ready to use
-```
-$ k get volumesnapshots demo-snapshot-podpvc -o yaml
-apiVersion: snapshot.storage.k8s.io/v1alpha1
-kind: VolumeSnapshot
-metadata:
-  creationTimestamp: 2018-10-05T16:59:26Z
-  generation: 1
-  name: demo-snapshot-podpvc
-  namespace: default
-
-...
-
-status:
-  creationTime: 2018-10-05T16:59:27Z
-  ready: true
-  restoreSize: 6Gi
-
-```
-4. Create a new volume from the snapshot
-```
-$ kubectl create -f ./examples/kubernetes/demo-restore-snapshot.yaml
-```
-
-## Kubernetes Development
-
-### Manual
-
-To build and install a development version of the driver:
-```
-$ GCE_PD_CSI_STAGING_IMAGE=gcr.io/path/to/driver/image:dev   # Location to push dev image to
-$ make push-container
-
-# Modify controller.yaml and node.yaml in ./deploy/kubernetes/dev to use dev image
-$ GCE_PD_DRIVER_VERSION=dev
-$ ./deploy/kubernetes/deploy-driver.sh
-```
-
-To bring down driver:
-```
-$ ./deploy/kubernetes/delete-driver.sh
-```
-
-## Testing
-
-Running E2E Tests:
-```
-$ PROJECT=my-project                               # GCP Project to run tests in
-$ IAM_NAME=my-iam@project.iam.gserviceaccount.com  # Existing IAM Account with GCE PD CSI Driver Permissions
-$ ./test/run-e2e-local.sh
-```
-
-Running Sanity Tests:
-```
-$ ./test/run-sanity.sh
-```
-
-Running Unit Tests:
-```
-$ ./test/run-unit.sh
-```
-
-## Dependency Management
-
-Use [dep](https://github.com/golang/dep)
-```
-$ dep ensure
-```
-
-To modify dependencies or versions change `./Gopkg.toml`
+### Kubernetes Beta Features
+
+* Topology: Requires K8s 1.14+ on Master and Nodes and PD driver v0.5.0+
```

Comment:
0.5.0 not yet release

Reply:
If I change it to "dev", are we going to remember to change it back?

Reply:
probably not, lets leave it like this